packages/http-client-csharp/src/type/operationLongRunning.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+import { OperationFinalStateVia } from "./operationFinalStateVia.js";
+import { OperationResponse } from "./operationResponse.js";
+
+export interface OperationLongRunning {
```

Comment:
Should this be part of azure-specific behaviors instead of the core client emitter library?

Reply:
For now everything will live here until we break them apart.  @bterlson I believe said this is fine as a temporary state.

Reply:
tracking item https://github.com/microsoft/typespec/issues/3114